resources/public/api/conf/P1/application.raml

```diff
@@ -25,29 +25,28 @@ uses:
   types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
```

Comment:
also need to change https://github.com/hmrc/individuals-employments-api/blob/master/conf/sandbox_version_P1.routes#L3

Reply:
and https://github.com/hmrc/individuals-employments-api/blob/a1f78f41b17af82876141431cadb73f09c486356/resources/public/api/conf/P1/examples/root.json#L4